### PR TITLE
Add CLI

### PR DESCRIPTION
**Pull request by denysdovhan**
I've removed `test/bench.js` and add CLI decs into `readme.md`. Please, check and merge if all right.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -53,6 +52,36 @@ tf = textr({ locale: 'ru'})
...
+# load through stdin, write result into file
+$ cat foo.md | textr -o bar.md
+```
+
+Take a look for more [usage examples][examples] and [CLI options][options].
+
+[cli]: https://github.com/denysdovhan/textr-cli
+[examples]: https://github.com/denysdovhan/textr-cli#usage
+[options]: https://github.com/denysdovhan/textr-cli#options
+
```
Comment (iamstarkov) [inline readme.md]: cli documention here will become outdate pretty fast, so maybe better just have this section `## CLI \n\n\n Also, there is a [CLI package][cli] for running Textr under your terminal.`. what do you think?

File: test/bench.js [hunk 2]
```diff
@@ -1,65 +0,0 @@
...
-  if (!program.cycles) { tf = textr().use.apply(null, tfs); }
-  do {
-    if (program.cycles) { tf = textr().use.apply(null, tfs); }
-    res = tf(buf);
-    i++;
-  } while (+new Date() < endedAt);
-  process.stdout.write(res);
-  process.stdout.write('\r\n');
-  process.stdout.write(i + ' cycles per second');
-}
```
Comment (iamstarkov) [inline test/bench.js]: 👍

**Comment by shuvalov-anton** [comment]
I see 2 different ways: 1) Move CLI to the core textr module that approach will resolve problem w/ outdated docs 2) put link to the CLI into the head part of the docs. So, what you take, what you'll give to your collaborators? :D

**Comment by shuvalov-anton** [comment]
@denysdovhan Oh, if you take first one, I'll give access to this repo for you, of course.

**Comment by shuvalov-anton** [comment]
@iamstarkov what you think about my idea?

**Comment by iamstarkov** [comment]
@shuvalov-anton i dont like idea of having cli and tool itself in one repo

**Comment by shuvalov-anton** [comment]
But it highly coupled to the another one.

**Comment by shuvalov-anton** [comment]
@iamstarkov anyway we have second point

**Comment by denysdovhan** [comment]
> i dont like idea of having cli and tool itself in one repo and in one package

I don't like too, bcoz would be better to keep textr core w/o deps.

File: readme.md (hunk 1 above)
Comment (denysdovhan) [inline readme.md]: I think base flags will stand unchanged in future, so we can keep it readme. Of course, will be new features but for this case there is line:

> Take a look for **more** [usage examples][examples] and [CLI options][options].

[examples]: https://github.com/denysdovhan/textr-cli#usage
[options]: https://github.com/denysdovhan/textr-cli#options

**Comment by shuvalov-anton** [comment]
Ok, if so, just take second one. @denysdovhan can you add just link and few words about CLI to the top of the readme file?

**Comment by shuvalov-anton** [comment]
@iamstarkov @denysdovhan  please decide what we should have in the readme file. I've checked current pr just now, and it looks ok to me. I hope that the basic examples should be constant in the long term. So I'm ok to merge it.

**Comment by denysdovhan** [comment]
> can you add just link and few words about CLI to the top of the readme file?

What do you mean? Move my changes to the top or what?

**Pull request closed without merging**